jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -101,9 +104,43 @@ public Throwable getThrowable() {
 	
 	private static native boolean isCheckpointAllowed();
 
-	private static native CRIUResult checkpointJVMImpl(String checkPointDir, boolean keepRunning, boolean shellJob, boolean extUnixSupport, int logLevel, String logFile);
+	private static native CRIUResult checkpointJVMImpl(String imagesDir,
+                                                       boolean leaveRunning,
+                                                       boolean shellJob,
+                                                       boolean extUnixSupport,
+                                                       int logLevel,
+                                                       String logFile,
+                                                       boolean fileLocks,
+                                                       String workDir);
 
-	private CRIUSupport() {}
+	/**
+	 * Constructs a new {@code CRIUSupport}.
+	 * 
+	 * The default CRIU dump options are:
+	 * {@code imagesDir} = null (must be set)
```

Comment:
I would put `{@code imagesDir} = imageDir`

Reply:
What do you mean by this? The default directory is `imageDir`?

Reply:
Nvm I see what you mean (the parameter)